hardware_interface:
  loop_hz: 10
  joint: joint_1